assets/js/navigation.js

```diff
@@ -150,17 +150,117 @@ function guideNavClick(id) {
 	}
 }
 
-jQuery( document ).ready(function( $ ) {
-	// We use JS to add a span that is used to tap on (mobile only) to shew the sub-menu, and is hidden by CSS on Desktop.
+function navBarOptimization() {
```

Comment:
I can't see where `guideNavClick` is used, I can't find a reference to it anywhere else in the Hale codebase.

Reply:
As `event` is used within the function, it is best to have it passed in, so I've changed it thus.  However, I suspect this might be a hang-on from Nightingale so maybe the whole function should go?